src/utils/ObjectUtils.js

```diff
@@ -0,0 +1,22 @@
+import * as R from "ramda";
+
+const mergeDeepRightObjectArrays = (leftValue, rightValue) => {
+  if(!Array.isArray(leftValue) || !Array.isArray(rightValue)) {
+    return R.clone(rightValue);
+  }
+
+  const mergedArray = R.clone(leftValue);
+  rightValue.forEach((value , index) => {
+    mergedArray[index] = R.clone(value);
```

Comment:
```suggestion
    mergedArray[index] = R.mergeDeepRight(mergedArray[index], value);
```

Reply:
I use our mergeDeepRight instead of clone now